test/features/card-input-modifier/index.test.ts

```diff
@@ -1,20 +1,30 @@
-import bus from "framebus";
-import CardInputModifier from "Features/deck-builder-features/card-input-modifier";
-import deckParser from "Lib/deck-parser";
-import scryfall from "Lib/scryfall";
-import mutation from "Lib/mutation";
+import * as bus from "framebus";
+import CardInputModifier from "../../../src/js/features/deck-builder-features/card-input-modifier";
+import deckParser from "../../../src/js/lib/deck-parser";
```

Comment:
Any special reason these are not using the aliases? Maybe you were working on a branch before I added them to the tsconfig?

https://github.com/crookedneighbor/shambleshark/blob/master/tsconfig.json#L5-L16

Reply:
I think so. I'll look through the files and adjust them, and see if my IDE still does these wonky imports.